Dockerfile

```diff
@@ -39,8 +39,19 @@ RUN yum install -y \
   ant \
   which\
   wget \
-  expect && \
-  yum groupinstall -y "Development Tools"
+  expect \
+  zlib-devel \
+  openssl-devel && \
+  yum groupinstall -y "Development Tools" && \
+  yum clean all
+
+#install ruby and fastlane
+RUN wget https://cache.ruby-lang.org/pub/ruby/2.5/ruby-2.5.1.tar.gz && \
+tar -zxvpf ruby-2.5.1.tar.gz && \
+(cd ruby-2.5.1; ./configure; make; make install;) && \
+gem update --system && \
+gem install fastlane -v ${FASTLANE_DEFAULT_VERSION} && \
```

Comment:
let's install it without documentation for efficiency:
`gem install --no-rdoc --no-ri ...`